general_rules/Scenario.tex

```diff
@@ -1,202 +1,122 @@
 	\end{enumerate}
 
-	\item \textbf{\iterm{Unknown objects}:} Any other object that is not known beforehand but can be grasped or handled.
+	\item \textbf{\UnknownObjects{}:} Any other object that is not known beforehand but can be grasped or handled (e.g. arena decorations).
 \end{enumerate}
 
-\paragraph*{Remark:} Most (known) objects used in the competition are chosen from the YCB dataset (see {\small\url{http://www.ycbbenchmarks.com/object-set/}}).
-These objects are considered standard, and can be acquired and trained in advance.
-Additional information about which \iterm{standard objects} were chosen, their official names, categories, and how to acquire them, can be found in the RoboCup@Home Official Website {\small\url{https://athome.robocup.org/standard-objects}} no later than 6 months prior to the \YEAR{} competition.
-
-\subsection{List of Predefined Objects}
-\label{rule:scenario_objects_list}
-The minimal configuration consists of:
+Known objects include at least:
 \begin{itemize}
-	\item \textbf{\iterm{Tableware}:} Dish, bowl, cup (or mug), and napkin.
+	\item \textbf{\iterm{Tableware}:} Dish, bowl, cup, and napkin (See~\reffig{fig:scenario_container_bowl}).
 	\item \textbf{\iterm{Cutlery}:} Fork, knife, and spoon.
 	\item \textbf{\iterm{Trash Bags}:} Big plastic trashbags, preferrably with handle.
-	\item \textbf{\iterm{Bags}:} Lightweight. With stiff, vertical handles.
-	\item \textbf{\iterm{Disks or books}:} A set of 10 discs (LP, CD, DVD, or BluRay) or books, all of the same kind.
-	\item \textbf{\iterm{Coat rack}:} A rack or pole to hang coats and other clothes.
-	\item \textbf{\iterm{Trays}:} A transport object like a tray or basket. Intended for two-handed manipulation.
-	\item \textbf{\iterm{Pourable}:} An object whose content can be poured (e.g.~muesli, cereal, etc.).
-	\item \textbf{\iterm{Heavy object}:} Weight between 1.0kg and 1.5kg.
-	\item \textbf{\iterm{Tiny object}:} A lightweight object with no bigger than 5cm (e.g.~paper, teabag, pen).
-	\item \textbf{\iterm{Fragile object}:} An easy-to-break object, (e.g.~chocolate egg).
-	\item \textbf{\iterm{Amorphous object}:} An flexible object that may take an infinite number of shapes (e.g.~cloth, magnetic puzzle, etc.).
-	\item \textbf{\iterm{Garbage bag}:} A tie-able garbage bag.
+	\item \textbf{\iterm{Bags}:} Lightweight. With stiff, vertical handles (See~\reffig{fig:scenario_container_bag}).
+	\item \textbf{\iterm{Trays}:} Tray or basket, intended for two-handed manipulation (See~\reffig{fig:scenario_container_tray}).
+	\item \textbf{\iterm{Pourable}:} An object whose content can be poured (e.g.~jug).
+	\item \textbf{\iterm{Heavy Object}:} Weight between 1.0kg and 1.5kg (e.g.~water bottle).
+	\item \textbf{\iterm{Tiny Object}:} A lightweight object, no bigger than 5cm (e.g.~teabag).
+	\item \textbf{\iterm{Fragile Object}:} An easy-to-break object (e.g.~egg).
+	\item \textbf{\iterm{Amorphous Object}:} A flexible object that may appear in different shapes (e.g.~cloth).
```

Comment:
The standard term here would be "deformable." Random google result exemplifying common usage: https://arxiv.org/abs/1910.13439

Reply:
Changed